faststream/__about__.py

```diff
@@ -1,5 +1,5 @@
 """Simple and fast framework to create message brokers based microservices"""
-__version__ = "0.1.5"
+__version__ = "0.1.6"
```

Comment:
Why we are incrementing version if there no any changes in `faststream/` directory?